opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -449,9 +450,10 @@ namespace Opm
                                        const GroupState& group_state,
                                        DeferredLogger& deferred_logger)
     {
+        // try to regularize equation if the well does not converge
+        const Scalar regularization_factor =  this->regularize_? this->param_.regularization_factor_ms_wells_ : 1.0;
+        const double volume = 0.002831684659200 * regularization_factor; // 0.1 cu ft;
```

Comment:
Since you are changing this line anyway: should use the unit support for this.
```
const double volume = 0.1 * unit::cubic(unit::feet);
```

Reply:
There is nothing like trivial fixes. This change triggered test failures.

Reply:
> There is nothing like trivial fixes. This change triggered test failures.

That is surprising! I assume you mean *more* test failures (compared to before)?

Reply:
I did some more testing and I can confirm that this change does not change anything as expected. I also did a rebase, so something else probably changed.